docs-v2/integrations/all/whatsapp-business.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Whatsapp Business
```

Comment:
WhatsApp

Reply:
The standard WhatsApp doesn’t officially offer a REST API for general or business use. Only the WhatsApp Business API, designed specifically for business interactions, provides a REST interface.

Reply:
sorry I meant the A in App

Reply:
I am sorry for missing that 🤦‍♂️, all good now fixed with [1bcc238](https://github.com/NangoHQ/nango/pull/2903/commits/1bcc238d57449631627b1aa162415cff24438b53)